ml-agents-envs/mlagents/envs/sampler_class.py

```diff
@@ -19,13 +19,19 @@ class UniformSampler(Sampler):
     """
 
     def __init__(
-        self, min_value: Union[int, float], max_value: Union[int, float], **kwargs
+        self,
+        min_value: Union[int, float],
+        max_value: Union[int, float],
+        seed: Optional[int] = None,
```

Comment:
I do not think you can have optional fields followed by **kargs. If you do then every time there is **kargs you must provide a seed.

Reply:
I think default parameters followed by **kwargs are OK. But they have to be after *args. 

https://www.digitalocean.com/community/tutorials/how-to-use-args-and-kwargs-in-python-3